docs/source/en/guides/collections.md

```diff
@@ -58,6 +58,45 @@ A `note` can also be attached to the item. This is useful to add additional info
 
 In addition to these base attributes, returned items can have additional attributes depending on their type: `author`, `private`, `lastModified`, `gated`, `title`, `likes`, `upvotes`, etc. None of these attributes are guaranteed to be returned.
 
+## List collections
+
+We can also get multiple collections with [`list_collection`]. It allows us to retrieve collections and filter them with certain parameters. Let's list all collections of the user `teknium`.
+```py
+>>> from huggingface_hub import list_collections
+
+>>> collections = list_collections(owner="teknium")
+```
+This returns an iterable of `Collection` objects. We can iterate over it to print number of upvotes for each collections.
+```py
+>>> for i in collections:
+...   print("Number of upvotes:", i.upvotes)
+Number of upvotes: 1
+Number of upvotes: 5
+```
+
+It is possible to do more advanced filtering. Let's get all model collections, sort by trending, and limit the count to 5.
+```py
+>>> collections = list_collections(item="models", sort="trending", limit=5):
+>>> for i in collections:
+...   print("Number of upvotes:", i.upvotes)
+Number of upvotes: 39
+Number of upvotes: 178
+Number of upvotes: 21
+Number of upvotes: 11
+Number of upvotes: 14
+```
+
+`sort` accepts one of the following values:
+* `last_modified`
+* `trending`
+* `upvotes`
+
+`item` accepts `spaces`, `models`, `datasets`, or `papers`. It is possible to specify a particular item, too. For example:
+* `models/teknium/OpenHermes-2.5-Mistral-7B`
+* `spaces/julien-c/open-gpt-rhyming-robot`
+* `datasets/squad`
+* `papers/2311.12983`
```

Comment:
```suggestion
Parameter `sort` must be one of  `"last_modified"`,  `"trending"` or `"upvotes"`. Parameter `item` accepts any particular item. For example:
* `"models/teknium/OpenHermes-2.5-Mistral-7B"`
* `"spaces/julien-c/open-gpt-rhyming-robot"`
* `"datasets/squad"`
* `"papers/2311.12983"`

For more details, please check out [`list_collections`] reference.
```

It looks like supporting generic search like `item="spaces"` is more a bug than a feature :confused: I will open an issue on the server repo to fix this. For example, if I go to https://huggingface.co/api/collections?sort=upvotes&item=spaces, the first collection listed is https://huggingface.co/collections/TheBloke/recent-models-last-100-repos-sorted-by-creation-date-64f9a55bb3115b4f513ec026 which contains only models... We will fix this but the safest is to document only the "search by specific item" use case.